metadata_mapper/mappers/oai/contentdm/arck_mapper.py

```diff
@@ -0,0 +1,32 @@
+from ..contentdm_mapper import ContentdmRecord, ContentdmVernacular
+
+class ArckRecord(ContentdmRecord):
+    @staticmethod
+    def identifier_match():
+        return ["view/ARCK3D"]
```

Comment:
I have a preference for static methods only because they offer a little more flexibility. A class attribute would work fine in this case. I think I might have over-engineered a bit here, hence the list rather than the string, because I thought that I saw examples of needing to match multiple strings to verify that the value is an identifier. 

I'm happy to use a class attribute and make this a single string rather than many.

Reply:
I was more curious to hear your principles than I was for a change. I don't think I have strong proclivities here. @barbarahui - what do you think?

Reply:
Yeah, I was also just curious to hear your thinking on it @lthurston . This makes sense to me. The class attribute would be a nice one line statement, but I think this is fine as is (and consistent with analogous static methods belonging to other classes).

Reply:
I would say that there are many cases when a class attribute is totally sufficient, and this is one of those. If it's a value that one is sure isn't going to vary, or potentially become tangled in some more complex logic based on the context the script is executed in, then class attributes exist for that purpose, as we all know.

I was on the fence about this one because I had an inkling that the ArckRecord might also need to also search for the Contentdm `identifier_match()` value, thus the list that I thought might include that parent class `identifier_match` value, which is easier to implement in a DRY way with a method. Having that logic bundled into the "getting" of that value feels tidy to me, rather than having it happen somewhere else.

I have no attachment to the approach here in this situation. It's unnecessary in retrospect. I agree one line is better, and will make that change before requesting approval on this PR again. Replacing the list with a string also simplifies some of the list comprehension code that was borderline hard to read.